api/dps/server.go

```diff
@@ -79,6 +81,11 @@ func (s *Server) GetLast(_ context.Context, _ *GetLastRequest) (*GetLastResponse
 // server.
 func (s *Server) GetHeightForBlock(_ context.Context, req *GetHeightForBlockRequest) (*GetHeightForBlockResponse, error) {
 
+	err := validator.New().Struct(req)
```

Comment:
I would rather see the validator initialized once as a `validate` field as part of the `*Server` struct.